scripts/pylib/twister/twisterlib.py

```diff
@@ -2912,6 +2923,12 @@ def run(self):
             if instance.handler.type_str == "device":
                 instance.handler.suite = self.suite
 
+            if(self.seed is not None and instance.platform.name.startswith("native_posix")):
+                self.parse_generated()
+                if('CONFIG_FAKE_ENTROPY_NATIVE_POSIX' in self.defconfig and
```

Comment:
hmm, not sure about that, dont you want to always enable fake entropy when running twister with --seed?